definitions/infra-kubernetes_statefulset/definition.yml

```diff
@@ -0,0 +1,25 @@
+domain: INFRA
+type: KUBERNETES_STATEFULSET
+configuration:
+  entityExpirationTime: DAILY
+  alertable: true
+
+synthesis:
+  identifier: entity.id
+  name: k8s.statefulsetName
+  encodeIdentifierInGUID: false
+  conditions:
+  - attribute: k8s.statefulset.podsDesired
+  tags:
+    k8s.clusterName:
+    k8s.namespaceName:
+    k8s.statefulsetName:
+    newrelic.integrationName:
+    newrelic.integrationVersion:
+      multiValue: false
+    newrelic.agentVersion:
+      multiValue: false
+    aws.region:
+      multiValue: false
+    aws.ec2.instanceType:
```

Comment:
I'm a bit lost on why these are not multiValue

Couldn't a statefulset be on different instance types and regions?

Reply:
Link to explanations in another PR: https://github.com/newrelic-experimental/entity-synthesis-definitions/pull/136/files#r604058166

Reply:
Also: a statefulset doesn't run anywhere in any region or instance. It creates pods, that will run somewhere.